app/org/sagebionetworks/bridge/models/accounts/UserSessionInfo.java

```diff
@@ -41,6 +42,7 @@ public UserSessionInfo(UserSession session) {
         this.sharingScope = session.getUser().getSharingScope();
         this.username = session.getUser().getUsername();
         this.roles = session.getUser().getRoles();
+        this.dataGroups = session.getUser().getDataGroups();
```

Comment:
Minor: If User.getDataGroups() is immutable, it might not be necessary to create an immutable copy here. That said, it would be best practice to do so anyway, in case the semantics of User changes somewhere down the line. Besides, ImmutableSet.copyOf() has optimizations which should make this a relative cheap operation.